ui/src/actionBar.tsx

```diff
@@ -21,6 +21,9 @@ export default function ActionBar(props: ActionBarProps) {
 
   const location = useLocation();
 
+  const [backURL, set] = useState<string | undefined>();
```

Comment:
you just need `<string>` here and the types will include undefined because you have no default value